### PR TITLE
Changed MD5 to SHA256 for FIPS compliance

### PR DESCRIPTION
**Pull request by brian-phantom**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [x] Testing of all the changes has been performed (for bug fixes / features)
- [x] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [x] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [x] Verify all checks are passing.
- [x] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [x] Do NOT submit updates to dependencies unless it fixes an issue.
- [x] Limit your pull request to one application and submit multiple pull requests if necessary.
#### Please check the type of change your PR introduces:
- [ ] New App
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

File: netskope.json [hunk 1]
```diff
@@ -10,12 +10,14 @@
     "product_version_regex": ".*",
     "publisher": "Netskope",
     "license": "Copyright (c) 2018-2020 Splunk Inc.",
```
Comment (rgil-splunk) [inline netskope.json]: copyright year needs to be updated. not sure why the copyright year test in static tests test didn't run 🤔

File: netskope_connector.py [hunk 2]
```diff
@@ -192,7 +197,8 @@ def _make_scim_rest_call(self, endpoint, action_result, params=None, timeout=60,
                 self._scim['token'] = config[NETSKOPE_CONFIG_SCIM_KEY]
             except Exception as e:
                 self.debug_print('Error while encoding server URL')
-                return RetVal(action_result.set_status(phantom.APP_ERROR, ('Error while encoding server URL: {}').format(e)), resp_json)
+                return RetVal(action_result.set_status(phantom.APP_ERROR,
+                        ('Error while encoding server URL: {}').format(e)), resp_json)
```
Comment (rgil-splunk) [inline netskope_connector.py]:
```suggestion
                        'Error while encoding server URL: {}'.format(e)), resp_json)
```

File: netskope_connector.py [hunk 3]
```diff
@@ -2103,22 +2156,22 @@ def finalize(self):
...
             headers['Referer'] = login_url
-            print 'Logging into Platform to get the session id'
-            response2 = requests.post(login_url, verify=False, data=data, headers=headers)
+            print('Logging into Platform to get the session id')
+            response2 = requests.post(login_url, verify=False, data=data, headers=headers)  # nosemgrep
             session_id = response2.cookies['sessionid']
         except Exception as e:
-            print ('Unable to get session id from the platform. Error: {}').format(str(e))
+            print('Unable to get session id from the platform. Error: {}'.format(str(e)))
             exit(1)
```
Comment (rgil-splunk) [inline netskope_connector.py]:
```suggestion
            sys.exit(1)
```

File: netskope_connector.py [hunk 4]
```diff
@@ -2103,22 +2156,22 @@ def finalize(self):
...
+        print(json.dumps(in_json, indent=4))
         connector = NetskopeConnector()
         connector.print_progress_message = True
         if session_id is not None:
             in_json['user_session_token'] = session_id
             connector._set_csrf_info(csrftoken, headers['Referer'])
         ret_val = connector._handle_action(json.dumps(in_json), None)
-        print json.dumps(json.loads(ret_val), indent=4)
+        print(json.dumps(json.loads(ret_val), indent=4))
     exit(0)
```
Comment (rgil-splunk) [inline netskope_connector.py]:
```suggestion
    sys.exit(0)
```

**Review by rgil-splunk — APPROVED** [review]

**Review by ricardo-phantom — APPROVED** [review]

**Pull request merged**